### ISSUE TITLE
Customizing/Manage settings of ublock with Firefox 61

### ISSUE THREAD

**Issue by florentlagoda** [uBlockOrigin/uBlock-issues#96]
### Prerequisites

- [X] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [ ] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [X] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- I tried to reproduce the issue when...
    - [X] uBlock Origin is the only extension
    - [ ] uBlock Origin with default lists/settings
    - [X] using a new, unmodified browser profile
- [X] I am running the latest version of uBlock Origin
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### Description

I try to manage white list of ublock to add our domains on Firefox 61 (64 bits) but it's not work.

I use this docs:
https://github.com/gorhill/uBlock/wiki/Deploying-uBlock-Origin
http://raymondhill.net/ublock/adminSetting.html

In about:config i add new String "extensions.ublock0.adminSettings" with this value :
"{\"netWhitelist\":\"about-scheme\\nchrome-extension-scheme\\nchrome-scheme\\nmoz-extension-scheme\\nopera-scheme\\nuhb.fr\\nuniv-rennes2.fr\\nvivaldi-scheme\\nwyciwyg-scheme\"}"

I also tried as a plain string value like this :
"{\"netWhitelist\":\"about-scheme\\nchrome-extension-scheme\\nchrome-scheme\\nmoz-extension-scheme\\nopera-scheme\\nuhb.fr\\nuniv-rennes2.fr\\nvivaldi-scheme\\nwyciwyg-scheme\"}"

I restart Firefox but our domains uhb.fr and univ-rennes2.fr not appears in the white-list and ublock stay active in our domains.

### A specific URL where the issue occurs

[A specific URL is MANDATORY for issue happening on a web page, even if it happens "everywhere"]

### Steps to Reproduce

1. In about:config i add new String "extensions.ublock0.adminSettings" with this value :
"{\"netWhitelist\":\"about-scheme\\nchrome-extension-scheme\\nchrome-scheme\\nmoz-extension-scheme\\nopera-scheme\\nuhb.fr\\nuniv-rennes2.fr\\nvivaldi-scheme\\nwyciwyg-scheme\"}"
2. Restart Firefox and go to univ-rennes2.fr
3. Ublock stay active and not appear in the white-list

### Expected behavior:

Domains uhb.fr and univ-rennes2.fr appears in the white-list and ublock not active on this domains

### Actual behavior:

Domains uhb.fr and univ-rennes2.fr not appears in the white-list and ublock active on this domains

### Your environment

* uBlock Origin version: 1.16.10
* Browser Name and version: Firefox 61 - 64 bits
* Operating System and version: Windows 7 64 bits and Windows 10 1709/1803 64 bits

**Comment by gwarser** [comment]
Did you check "Further readings" section? https://github.com/gorhill/uBlock/issues/2986#issuecomment-364035002

**Comment by gorhill** [comment]
Not a bug, `about:config` approach is deprecated since uBO is a webext.

**Comment by gorhill** [comment]
The wiki page contains this passage:

> for uBO WebExtension, Firefox 57+ read https://github.com/gorhill/uBlock/issues/2986#issuecomment-333198882

**Comment by gwarser** [comment]
@gorhill I added this 30 minutes ago :) Will try to transfer information from #2986 to the wiki later.

**Comment by gorhill** [comment]
> I added this 30 minutes ago

Ak ok, sorry to @florentlagoda then.

Thinking of it, I think it's best that admins with 1st-hand experience on how to solve such issues contribute directly to the wiki page. I can't test anything related to this, and thus it end up merely trying to convey in there what somebody else with 1st hand experience mentioned somewhere.

**Comment by gwarser** [comment]
Yes, I agree. If I find some documentation I will add a link, otherwise [...]

I see @uBlock-user did everything I wanted to do, thank you.

**Comment by uBlock-user** [comment]
@gwarser Feel free to expand on Firefox, as I have only linked to the comment instead of offering an indepth explanation, since I'm not on much aware of the how tos regarding this.
